aldryn_config.py

```diff
@@ -4,8 +4,8 @@
 
 class Form(forms.BaseForm):
 
-    show_all_recipients = forms.CheckboxField('Show all users in form recipients select', required=False)
+    show_all_recipients = forms.CheckboxField('Include only admin users as selectable e-mail recipients instead of everyone.', required=False)
 
     def to_settings(self, data, settings):
-        settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = data['show_all_recipients']
+        settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = not data['show_all_recipients']
```

Comment:
First off ``settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = not data['show_all_recipients']`` looks very wrong. If you are going to invert the logic, better change the name of the variable as well. So change both the setting and the form field name to actually express the right thing.

If you rename the form field, the next time someone accesses the form, it will just not have any data (and in the case of checkbox will default to false). There is no way to migration or anything like that.
You could make the new field a ``SelectField`` that is required. So the user would get a validation error and needs to take another look at the option. But that would be a bit weird if the only options are "yes" and "no".